app/src/ui/diff/text-diff.tsx

```diff
@@ -137,7 +165,11 @@ const defaultEditorOptions: IEditorConfigurationExtra = {
   lineWrapping: true,
   mode: { name: DiffSyntaxMode.ModeName },
   // Make sure CodeMirror doesn't capture Tab (and Shift-Tab) and thus destroy tab navigation
-  extraKeys: { Tab: false, 'Shift-Tab': false },
+  extraKeys: {
+    Tab: false,
+    'Shift-Tab': false,
+    [__DARWIN__ ? 'Cmd-F' : 'Ctrl-F']: showSearch,
```

Comment:
i literally didn't know you could do this with a ternary

Reply:
It comes in pretty handy at times, see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Computed_property_names

I was sure we used it elsewhere in the app but during my quick look for examples I didn't find anything.